bin/tblextr.py

```diff
@@ -449,6 +494,8 @@ def fill_ops_db(table_name, db, indir):
   hsa_statfile = re.sub(r'\.stats\.csv$', r'.hsa_stats.csv', statfile)
   hip_statfile = re.sub(r'\.stats\.csv$', r'.hip_stats.csv', statfile)
   kfd_statfile = re.sub(r'\.stats\.csv$', r'.kfd_stats.csv', statfile)
+  sysinfo_file = re.sub(r'\.stats\.csv$', r'.sysinfo_stats.txt', statfile)
+  params = json_metadata_gen(sysinfo_file, 1)
```

Comment:
Don;t need to return anything
Should just append info to file
Instead of integer the command line should be a parameter:
metadata_gen(sysinfo_file, '/opt/rocm/bin/rocminfo')

Reply:
Done